docker-compose.yml

```diff
@@ -38,6 +38,7 @@ x-op-backend: &backend
   environment:
     LOCAL_DEV_CHECK: "${LOCAL_DEV_CHECK:?The docker-compose file for OpenProject has moved to https://github.com/opf/openproject-deploy}"
     RAILS_ENV: development
+    OPENPROJECT_HOST__NAME: "${HOST:-localhost}:${PORT:-3000}"
```

Comment:
@machisuji Should this be moved to override example?

Reply:
Quite possibly. I will give it a go in a bit.

Reply:
I moved it, as it feels odd to set environment variable for setting if it supposed to be changed in override file